### PR TITLE
feature: health-check artisan command

### PR DESCRIPTION
**Pull request by marcoocram**

### CODE REVIEW

File: tests/Console/HealthCheckCommandTest.php [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+use Letsgoi\HealthCheck\HealthCheck;
+use Letsgoi\HealthCheck\Tests\Stubs\FakeOkChecker;
+use Letsgoi\HealthCheck\Tests\TestCase;
+use Mockery\MockInterface;
+
+class HealthCheckCommandTest extends TestCase
+{
+    /** @test */
+    public function it_should_healt_check_with_all_checkers_on_command()
```
Comment (abalozz) [inline tests/Console/HealthCheckCommandTest.php:13]: There is a little typo

```suggestion
    public function it_should_health_check_with_all_checkers_on_command()
```

File: tests/Console/HealthCheckCommandTest.php [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+                ->once()
+                ->andReturn(true);
+        });
+
+        $this->artisan('health-check')
+            ->expectsOutput('All checkers are ok.');
+    }
+
+    /** @test */
+    public function it_should_healt_check_with_one_checker_on_command()
```
Comment (abalozz) [inline tests/Console/HealthCheckCommandTest.php:26]: There is a little typo
```suggestion
    public function it_should_health_check_with_one_checker_on_command()
```

File: tests/Console/HealthCheckCommandTest.php (hunk 1 above)
Comment (marcoocram) [reply tests/Console/HealthCheckCommandTest.php]: I've serious problems with this word..... it's fixed

File: tests/Console/HealthCheckCommandTest.php (hunk 2 above)
Comment (marcoocram) [reply tests/Console/HealthCheckCommandTest.php]: I've serious problems with this word..... it's fixed

**Pull request merged**